website/server/models/group.js

```diff
@@ -446,6 +447,33 @@ schema.methods.startQuest = async function startQuest (user) {
   });
 };
 
+schema.methods.sendGroupChatReceivedWebhooks = function sendGroupChatReceivedWebhooks (chat) {
+  let query = {
+    webhooks: {
+      $elemMatch: {
```

Comment:
Talked in slack with @paglias about using the shorthand for this instead, but I think for this case it needs to be `$elemMatch`, because its matching on both the `type` and the `options.groupId`